frontend/src/pages/landing/Landing.loader.tsx

```diff
@@ -14,8 +14,11 @@ export interface LandingContext {
 }
 
 export const landingLoader = loginRequired(
-  async (): Promise<LandingContext> => {
-    const statusMap = await getStatusMap();
+  async ({ request }): Promise<LandingContext> => {
+    const url = new URL(request.url);
+    const queryParams = url.searchParams;
+    const orderQuery = queryParams.get("ordering");
+    const statusMap = await getStatusMap(orderQuery);
```

Comment:
Can we use a default ordering?

Reply:
We're already using a default ordering, no? `Landing.tsx` at line 207:
`const selectedSort =
    new URLSearchParams(window.location.search).get("ordering") || "-created";`

I think it's more appropriate to have this in the UI layer, or do you think differently?